lib/jhove.rb

```diff
@@ -55,7 +55,7 @@ def errors_for_field(field)
   private
 
   def feed_validate_script
-    components = ['perl', @config[:feed_validate_script], 'google mdp', @dir]
+    components = ["docker run --rm -v '#{@dir}:/images' ghcr.io/hathitrust/feed perl /usr/local/feed/bin/validate_images.pl google mdp /images"]
```

Comment:
Might need to mount at the dir's name... Something like this might help:

```
require 'pathname'
local = Pathname.new(@dir)
dir = local.basename
components = ["docker run --rm -v '#{local}:/images/#{dir}' ghcr.io/hathitrust/feed perl /usr/local/feed/bin/validate_images.pl google mdp /images/#{dir}"]
```

Reply:
Yes, this is allowing the Perl to run when run outside Docker. I'm working with a patch to handle the "fake feed" in test/bin so it doesn't try to run Docker inside Docker. See below for comment re namespace issue.